test/Swashbuckle.AspNetCore.Examples.Test/Examples/ExamplesOperationFilterTests.cs

```diff
@@ -115,20 +116,22 @@ public void Apply_WhenPassingDictionary_ShouldSetExampleOnRequestSchema()
         private void SetSwaggerResponses(Operation operation, OperationFilterContext filterContext)
         {
             var swaggerResponseFilter = new AnnotationsOperationFilter();
+            while (false == Debugger.IsAttached) { }
```

Comment:
Don't think we need this?

Reply:
uhhhh

Reply:
yeah, I added that to be able to debug a null-ref